src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -229,13 +228,13 @@ public Set<Table> getTablesForKeyspace(String keyspace) throws ReaperException {
   }
 
   @Override
-  public int getPendingCompactions() throws JMException {
+  public int getPendingCompactions() throws ReaperException {
     return 1; // TODO: implement me.
   }
 
   @Override
```

Comment:
We discussed this before, but just recording here so you have something to refer back to. I think the way this works in the JMX implementation is to go to Cassandra's JMX metrics registry and draw whether a repair is in progress from there.

The problem with just using job tracker, is that (AFAIK) it only tracks repairs initiated by this instance of Reaper. A repair started elsewhere (e.g. via nodetool) might not make its way into the job tracker, which means that this logic might miss running repairs. 

I believe that @adejanovski has been doing a lot of work around getting metrics from the new metrics endpoint here, so it might be worth syncing with him to discuss how he's done this, you may need to wait on some refactoring he's doing to the `MetricsProxy`.